packages/wasm-utils/Cargo.toml

```diff
@@ -24,11 +24,13 @@ ark-ff = { version = "^0.3.0", default-features = true }
 ark-serialize = "0.3.0"
 ark-crypto-primitives = { version = "^0.3.0", default-features = true, features = ["r1cs"] }
 ark-ed-on-bn254 = { version = "^0.3.0", default-features = true, features = ["r1cs"] }
+ark-bn254 = { version = "^0.3.0", default-features = true, features = ["curve"] }
+ark-bls12-381 = { version = "^0.3.0", default-features = false, features = ["curve"] }
 webb-pedersen-hash = { version = "0.1.0", git = "https://github.com/webb-tools/webb-rs.git" }
 ark-std ={ version = "^0.3.0" }
-arkworks-gadgets = { version = "^0.4.14", features = ["std", "r1cs"], default-features = true }
-arkworks-circuits = { version = "^0.4.14", features = ["std", "r1cs"], default-features = false }
-arkworks-utils = { version = "^0.4.14", features = ["std", "r1cs"], default-features = false }
+arkworks-gadgets = { version = "0.4.18", default-features = true }
+arkworks-circuits = { version = "0.4.18", default-features = false }
+arkworks-utils = { version = "0.4.18", default-features = false }
```

Comment:
Pegged to exact version. We haven't figured out a proper versioning process yet, so its safer this way